lib/matplotlib/backend_bases.py

```diff
@@ -400,7 +400,7 @@ def _iter_collection_uses_per_path(self, paths, all_transforms,
         if Npaths == 0 or (len(facecolors) == 0 and len(edgecolors) == 0):
             return 0
         Npath_ids = max(Npaths, len(all_transforms))
-        N = max(Npath_ids, len(offsets))
+        N = max(Npaths, len(offsets))
```

Comment:
(Also, if we put a fix there, I'd also just replace the next line by `return int(np.ceil(N / Npath_ids))` which is much clearer in intent.)